### PR TITLE
Enabling OPC Events e2e tests

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

**Review by cristipogacean — APPROVED** [review]
:shipit:

**Review by karok2m — APPROVED** [review]

**Comment by karok2m** [comment]
@dacolgit, can you please also run the Standalone E2E pipeline and post a link here ? Let's make sure that it is passing without any issues.

File: e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs [hunk 1]
```diff
@@ -106,27 +106,27 @@ public class IIoTPlatformTestContext : IDisposable, IDeviceConfig, IIoTHubConfig
         /// <summary>
         /// Urls for the dynamic ACI containers
         /// </summary>
-        public string[] PlcAciDynamicUrls { get; set; }
```
Comment (karok2m) [inline e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:109]: What were the build warnings here ? With nullable values we may need to start checking if they actually have value. Might be better if we can avoid that.

Comment (dacolgit) [reply e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:109]: these for example are the warnings:

image

Comment (karok2m) [reply e2e-tests/IIoTPlatform-E2E-Tests/TestExtensions/IIoTPlatformTestContext.cs:109]: Understood after discussing this with @dacolgit. This uses the same pattern as other similar variables in `IIoTPlatformTestContext` class.

**Review by monikavar — APPROVED** [review]

**Pull request closed without merging**